packages/p2p-v2/src/components/OnlineStatus/__tests__/OnlineStatusLabel.spec.tsx

```diff
@@ -17,8 +17,8 @@ jest.mock('@/utils', () => ({
 
 describe('<OnlineStatusLabel/>', () => {
     it('should call the getLastOnlineLabel function with isOnline and lastOnlineTime', () => {
-        render(<OnlineStatusLabel isOnline={0} lastOnlineTime={1685446791} />);
-        expect(getLastOnlineLabel).toHaveBeenCalledWith(0, 1685446791);
+        render(<OnlineStatusLabel isOnline={false} lastOnlineTime={1685446791} />);
```

Comment:
If we use the query above 👀 can remove this

```suggestion
        render(<OnlineStatusLabel lastOnlineTime={1685446791} />);
```